platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java

```diff
@@ -362,6 +363,39 @@ public String getFileContent(@PathParam("machineId") String machineId,
         return machineManager.getMachine(machineId).readFileContent(path, startFrom, limit);
     }
 
+    /**
+     * Copies files between specified machines
+     *
+     * @param sourceMachineId
+     *         id of source machine
+     * @param sourcePath
+     *         path to resources in source machine
+     * @param targetMachineId
+     *         id of target machine
+     * @param targetPath
+     *         destination path in target machine
+     * @param noOverwrite
```

Comment:
It is ugly. I understand that it is copied from docker API but I don't agree to follow their API